pkg/helm/chartproxy/proxy_test.go

```diff
@@ -174,7 +175,9 @@ func TestProxy_IndexFile(t *testing.T) {
 				t.Error(err)
 			}
 			if tt.mergedFile != "" {
-				expectedIndexFile, err := repo.LoadIndexFile(tt.mergedFile)
+				data, err := ioutil.ReadFile(tt.mergedFile)
+				expectedIndexFile := &repo.IndexFile{}
+				err = yaml.UnmarshalStrict(data, expectedIndexFile)
```

Comment:
err is set on line 178 but overwritten in line 180 before before being test checked. Should probably add a second check for !nil after 178. Same is also true in the repos_test.go test case.

Reply:
Oh, thanks yeah missing a check.

Reply:
Done